tests/integration_tests/test_ignite.py

```diff
@@ -0,0 +1,40 @@
+from ignite.engine import Engine
+from mock import Mock
+from mock import patch
+import pytest
+
+import optuna
+from optuna.testing.integration import create_running_trial
+from optuna.testing.integration import DeterministicPruner
+from optuna import type_checking
+
+if type_checking.TYPE_CHECKING:
+    from typing import Iterable  # NOQA
+
+
+def test_ignite_pruning_handler():
+    # type: () -> None
+
+    def update(engine, batch):
+        # type: (Engine, Iterable) -> None
+
+        pass
+
+    trainer = Engine(update)
+
+    # The pruner is activated.
+    study = optuna.create_study(pruner=DeterministicPruner(True))
+    trial = create_running_trial(study, 1.0)
+
+    handler = optuna.integration.IgnitePruningHandler(trial, 'accuracy', trainer)
+    with patch.object(trainer, 'state', epoch=Mock(return_value=1), metrics={'accuracy': 1}):
+        with pytest.raises(optuna.structs.TrialPruned):
+            handler(trainer)
+
+    # # The pruner is not activated.
```

Comment:
Redundant `#` characters:
```suggestion
    # The pruner is not activated.
```

Reply:
Sorry for my careless mistake. I fixed it. https://github.com/pfnet/optuna/pull/561/commits/3a840e6d8da019e141e6226b30223a88ef7b2367